File: src/main/java/entity/Authentication.java
package entity;

import jakarta.persistence.*;
import lombok.*;
import entity.Customer;

import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class Authentication {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer authId;
    
    private String password;
    private LocalDateTime lastLogin;
    private Byte status = 1;
    
    @OneToOne
    @JoinColumn(name = "cust_id")
    private Customer customer;
}

File: DotNet_Backend/SafarnamaApplication/SafarnamaApplication/Models/Authentication.cs
using System;
using System.Collections.Generic;

namespace SafarnamaApplication.Models;

public partial class Authentication
{
    public int AuthId { get; set; }

    public int? CustId { get; set; }

    public string? Password { get; set; }

    public DateTime? LastLogin { get; set; }

    public sbyte? Status { get; set; }

    public virtual Customer? Cust { get; set; }
}
